<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_transformar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 197, 198);</string>
    </property>
    <property name="text">
     <string>TRANSFORMAR</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>135</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="txt_libra">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 197, 198);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_farhen">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 197, 198);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_milla">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 197, 198);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>202</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe MDL2 Assets</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Segoe MDL2 Assets</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Masa  ( libra - kilo )</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Segoe MDL2 Assets</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Temperatura (  Farhenheit - Celsius )</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Segoe MDL2 Assets</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Longitud (Millas - Kilometros )</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>135</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="txt_kilo">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 197, 198);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_celsius">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 197, 198);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_km">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 197, 198);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sistema imperial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sistema internacional </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
